docs/content/policy-reference.md

```diff
@@ -69,8 +69,14 @@ complex types.
 ### Objects
 | Built-in | Description |
 | -------- | ----------- |
-| <span class="opa-keep-it-together">`filtered := json.filter(object, paths)`</span> | `filtered` is the remaining data from `object` with only keys specified in `paths` which is an array or set of key paths. Each path may be a JSON string path or an array of path segments. For example: `json.filter({"a": {"b": "x", "c": "y"}}, ["a/b"]` will result in `{"a": {"b": "x"}}`). The `json` string `paths` may reference into array values by using index numbers. For example with the object `{"a": ["x", "y", "z"]}` the path `a/1` references `y` |
 | <span class="opa-keep-it-together">`value := object.get(object, key, default)`</span> | `value` is the value stored by the `object` at `key`. If no value is found, `default` is returned. |
+| <span class="opa-keep-it-together">`output := object.remove(object, keys)`</span> | `output` is a new object which is the result of removing the specified `keys` from `object`. `keys` must be either an array or set of keys. |
+| <span class="opa-keep-it-together">`output := object.union(objectA, objectB)`</span> | `output` is a new object which is the result of an asymmetric recursive union of two objects where conflicts are resolved by choosing the key from the left-hand object. For example: `object.union({"a": 1, "b": 2}, {"a": 7, "c": 3})` will result in `{"a": 1, "b": 2, "c": 3}`  |
```

Comment:
Perhaps being right-handed makes me assume the right-hand side would win. No strong opinion here, but I'm curious if you feel the opposite or if this just fell out of the implementation.

Reply:
Let's update the example to show the recursive behavior.

Reply:
I don't have strong opinions about which side takes precedence, I just followed how it was described in https://github.com/open-policy-agent/opa/issues/1617 I can rationalize either way as making sense.

Want me to swap em?

Reply:
Nah, it's fine. FWIW, Tim mentioned later in that issue that people expected the RHS to win: https://github.com/open-policy-agent/opa/issues/1617#issuecomment-520613777

Reply:
I'll swap them 😆